### PR TITLE
Django: upgrade to 4.2

### PR DESCRIPTION
**Pull request by humitos**

### CODE REVIEW

**Review by stsewd — APPROVED** [review]

File: readthedocsext/theme/templates/includes/crud/pagination.html [hunk 1]
```diff
@@ -12,15 +12,15 @@
     {% if display_page_links %}
       {% for page in pages %}
         {% if page %}
-          {% ifequal page page_obj.number %}
+          {% if page == page_obj.number %}
```
Comment (agjohnson) [inline readthedocsext/theme/templates/includes/crud/pagination.html:15]: I didn't even know ifequal was a tag, let alone remember writing this :upside_down_face:

**Review by agjohnson — APPROVED** [review]
Looks good! I'll set this to blocking until we're ready to merge 4.2 support.

File: readthedocsext/theme/templates/includes/crud/pagination.html (hunk 1 above)
Comment (ericholscher) [reply readthedocsext/theme/templates/includes/crud/pagination.html:15]: Support for if statements in Django was a big deal :D

**Comment by humitos** [comment]
The main PR was merged 👍🏼

**Pull request merged**